build/Unmanaged.props

```diff
@@ -48,8 +48,8 @@
     <UseOfAtl>Static</UseOfAtl>
     <UseOfMfc>false</UseOfMfc>
     <CharacterSet>Unicode</CharacterSet>
-    <PlatformToolset>v141</PlatformToolset>
-    <WindowsTargetPlatformVersion>10.0.14393.0</WindowsTargetPlatformVersion>
+    <PlatformToolset>v142</PlatformToolset>
+    <WindowsTargetPlatformVersion>10.0.18362.0</WindowsTargetPlatformVersion>
```

Comment:
>18362 [](start = 39, length = 5)

We're using an older version for aerobee (10.0.17763.0) iirc. It would be nice to synch these up.

Reply:
Sure thing!

Reply:
Discussed offline, we should actually upgrade Aerobee